locations/dict_parser.py

```diff
@@ -346,6 +347,8 @@ def parse(obj: dict) -> Feature:
         else:
             item["country"] = country
 
+        item["opening_hours"] = DictParser.get_first_key(obj, DictParser.hours_keys)
```

Comment:
Why? This will just lead to malformed opening hours in the output

Reply:
ouch, I added it for testing and forgot it there